src/rpc/handlers/AMMInfo.cpp

```diff
@@ -21,12 +21,44 @@
 
 #include "data/DBHelpers.h"
 #include "rpc/AMMHelpers.h"
+#include "rpc/Errors.h"
+#include "rpc/JS.h"
 #include "rpc/RPCHelpers.h"
 #include "rpc/common/MetaProcessors.h"
 #include "rpc/common/Specs.h"
+#include "rpc/common/Types.h"
 #include "rpc/common/Validators.h"
 
+#include <boost/json/array.hpp>
```

Comment:
Questionable list of includes.. but i guess that's the purpose of automating this.